README.md

```diff
@@ -4,7 +4,57 @@
 ![GitHub repo size](https://img.shields.io/github/repo-size/pyroteus/goalie)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-### Install
+Goalie provides goal-oriented mesh adaptation pipelines for solving partial differential equations (PDEs) using adapted meshes built on the Python-based finite element library [Firedrake](http://www.firedrakeproject.org/).  It runs a fixed point iteration loop for progressively solving time-dependent PDEs and their adjoints on sequences of meshes, performing goal-oriented error estimation, and adapting the meshes in sequence with a user-provided adaptor function until defined convergence criteria have been met. It is recommended that users are familiar with adjoint methods, mesh adaptation and the goal-oriented framework before starting with Goalie.
 
-In order to use Goalie's metric-based mesh adaptation functionality, you will need a bespoke Firedrake installation.
-See the instructions for the [Animate](https://github.com/pyroteus/animate) package for details on this.
+Goal-oriented mesh adaptation presents one of the clearest examples of the intersection between adjoint methods and mesh adaptation. It is an advanced topic, so it is highly recommended that users are familiar with adjoint methods, mesh adaptation and the goal-oriented framework before starting with Goalie.
```

Comment:
This repeats the above.

Reply:
I removed the second version. I also added a link to the pyroteus.github.io where you go into goal-oriented mesh adaptation given the suggestion for the user to familiarise themselves with it.